arch/arm64/boot/dts/broadcom/bcm2712-rpi-cm4io.dtsi

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: GPL-2.0
+
+i2c_csi_dsi0: &i2c0 { // Note: For CAM0 and DISP0 connectors
+	symlink = "i2c-10";
+};
+
+i2c_csi_dsi1: &i2c6 { // Note: For CAM1, DISP1, on-board RTC, and fan controller
```

Comment:
And seeing as CAM1 and DISP1 share an I2C port, if the above is true about the MIPI wiring, that means we can't run two identical cameras or displays on CM5 with CM4IO (fine on CM5IO).

Reply:
No, you've got it backwards: CAM0 and DISP0 shared DPHY0, CAM1 and DISP1 share DPHY1.

Reply:
On CM4IO, of course. CM5IO only has the two sockets.

Reply:
CAM0 and DISP0 only have 2 data lanes each on the CM4 pinout, so how have we got the 4 data lanes of second PHY on CM5 routed out?

I'll have to find a schematic....

Reply:
We don't - the 3rd and 4th data lanes of CAM0 and DISP0 are unconnected.

Reply:
Yes, I believe that's correct.